e2e/testcafe-devextreme/tests/dataGrid/keyboardNavigation/editOnKeyPress.screenshots.ts

```diff
@@ -62,3 +64,75 @@ const DATA_GRID_SELECTOR = '#container';
     await makeRowsViewTemplatesAsync(DATA_GRID_SELECTOR);
   });
 });
+
+test('Focused cell should not flick (T1206435)', async (t) => {
+  type TestWindow = (typeof window) & {
+    counter?: number;
+  };
+
+  const dataGrid = new DataGrid(DATA_GRID_SELECTOR);
+  const firstCell = dataGrid.getDataCell(0, 0).element;
+  const secondCell = dataGrid.getDataCell(1, 0).element;
+  const getFocusEventCount = ClientFunction(
+    () => (window as TestWindow).counter,
+  );
+
+  await ClientFunction(() => {
+    const testWindow = window as TestWindow;
+    testWindow.counter = 0;
+    (secondCell() as any as HTMLElement).addEventListener('focusin', () => {
+      testWindow.counter! += 1;
+    });
+  }, {
+    dependencies: { secondCell },
+  })();
+
+  await t.click(firstCell);
+
+  await t.pressKey('M');
+  await t.pressKey('enter');
+
+  await t.expect(secondCell.focused).ok();
+
+  const focusEventCount = await getFocusEventCount();
+  await t.expect(focusEventCount).eql(1);
+
+  await ClientFunction(() => {
+    delete (window as TestWindow).counter;
+  })();
+}).before(async () => {
+  await createWidget('dxDataGrid', ClientFunction(() => {
```

Comment:
The ```createWidget``` implemented with the ```ClientFunction``` usage.
So, the ```ClientFunction``` here doesn't make any sense. 
Let's get rid of it:

```typescript
await createWidget('dxDataGrid', () => {
    const data = [
      { value: 'data' },
      { value: 'data' },
    ];
    return {
      dataSource: new (window as any).DevExpress.data.CustomStore({
        load() {
          return Promise.resolve(data);
        },
        update() {
          return new Promise<void>((res) => {
            setTimeout(() => {
              res();
            }, 100);
          });
        },
      }),
      keyboardNavigation: {
        enabled: true,
        editOnKeyPress: true,
        enterKeyAction: 'moveFocus',
        enterKeyDirection: 'column',
      },
      editing: {
        mode: 'cell',
        allowUpdating: true,
        allowAdding: true,
        startEditAction: 'dblClick',
        refreshMode: 'reshape',
      },
      repaintChangesOnly: true,
    };
  });
```